### PR TITLE
detect if `fastlane env` is in issue.

### PR DESCRIPTION
**Pull request by hjanuschka**
also includes `.rubocop.yml` from fastlane

### CODE REVIEW

File: bot.rb [hunk 1]
```diff
@@ -100,6 +101,20 @@ def process_inactive(issue)
       end
     end
 
+    # Remind people to include `fastlane env`
+
+    def process_env_check(issue)
```
Comment (hjanuschka) [inline bot.rb]: this is the actual change, the rest is coz of rubocop

**Comment by KrauseFx** [comment]
This looks nice, however what if there is an issue about code signing that doesn't include `fastlane env`? Ideally we want both comments to be added, right?

**Comment by hjanuschka** [comment]
i thinkt currently both comments should be added.
or do you mean that we should combine bot message to one (aka el-cheapo danger?)

**Comment by KrauseFx** [comment]
No, 2 comments is fine, but we check if there has already been a comment before posting one, right?

**Comment by hjanuschka** [comment]
ahhh damn thats true, ok give me a few minutes

**Comment by hjanuschka** [comment]
@KrauseFx  ok done, any idea how i test this? - its kinda like blind-coding - relaying on rubocop 😨 

  * also added rubocop to Gemfile
  * Gemfile.lock updated


do you think it is better to combine them  in one message?

File: bot.rb [hunk 2]
```diff
@@ -33,8 +33,16 @@ def start
...
+      bot_actions = []
       process_inactive(issue)
-      process_code_signing(issue)
+
+      return if issue.comments > 0 # there maybe already some bot replys
+      bot_actions << process_code_signing(issue)
+      bot_actions << process_env_check(issue)
+
+      bot_actions.each do |bot_reply|
+        client.add_comment(SLUG, bot_reply[:issue], bot_reply[:message])
```
Comment (KrauseFx) [inline bot.rb]: We don't need to also have the `issue` inside the hash, we already have access to it in the `issue` variable 👍

File: bot.rb [hunk 3]
```diff
@@ -100,10 +108,21 @@ def process_inactive(issue)
...
+    # Remind people to include `fastlane env`
+
+    def process_env_check(issue)
+      body = issue.body + issue.title
+      unless body.include?("Loaded fastlane plugins")
+        puts "https://github.com/#{SLUG}/issues/#{issue.number} (#{issue.title}) seems to be missing env report"
+        body = []
+        body << "It seems like you have not included the output of `fastlane env`."
+        body << "To make it easier for us help you resolve this issue, please update the issue to include the output of `fastlane env` :+1:"
+        return { issue: issue, message: body.join("\n\n") }
```
Comment (KrauseFx) [inline bot.rb]: So just returning the `body.join` here is good enough 👍

**Comment by hjanuschka** [comment]
@KrauseFx  done

**Comment by KrauseFx** [comment]
Awesome, thanks @hjanuschka 👍

**Pull request merged**